quick-start/api/swagger/mocks.json

```
@@ -20,6 +20,236 @@
     "http"
   ],
   "paths": {
+    "/flows": {
+      "get": {
+        "tags": [
+          "flows"
+        ],
+        "summary": "Returns all flows",
+        "description": "Returns all flows",
+        "operationId": "findFlows",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/flow"
+              },
+              "default": [
+                {
+                  "name": "MappingFlow",
+                  "description": "Maps raw values to the canonical model",
+                  "identifier": "cts.collectionQuery('rawContent')",
+                  "steps": [
+                    {
+                      "type": "mapping",
+                      "name": "person-mapping1.json",
+                      "identifier": null,
+                      "retryLimit": 0,
+                      "options": {}
+                    }
+                  ]
+                }
+              ]
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "flow"
+        ],
+        "summary": "Create flow",
+        "description": "",
+        "requestBody": {
```

For POST requests, I believe the parameters array (line 99). See example below
        "parameters": [
          {
            "in": "body",
```suggestion
        "requestBody": {
```
            "name": "body",
            "description": "Pet object that needs to be added to the store",
            "required": true,
            "schema": {
              "$ref": "#/definitions/Pet"
            }
          }
        ],

---

request body for post/put have been moved to parameters and they are appearing in the swagger ui properly